graphql/resolvers.go

```diff
@@ -2527,6 +2527,15 @@ func (r *annotationResolver) UserCanModify(ctx context.Context, obj *restModel.A
 
 }
 
+func (r *annotationResolver) WebhookConfigured(ctx context.Context, obj *restModel.APITaskAnnotation) (*bool, error) {
+	t, err := r.sc.FindTaskById(*obj.TaskId)
+	if err != nil {
+		return utility.FalsePtr(), InternalServerError.Send(ctx, fmt.Sprintf("error finding task: %s", err.Error()))
```

Comment:
It should return `nil, error` if there's an error> we don't want to return a result if theres an error getting this value

Reply:
thanks, fixed